frame/staking/src/lib.rs

```diff
@@ -1329,6 +1396,10 @@ pub mod pallet {
 		Kicked(T::AccountId, T::AccountId),
 		/// The election failed. No new era is planned.
 		StakingElectionFailed,
+		/// Attempted to rebag an account.
+		///
+		/// If the second parameter is not `None`, it is the `(from, to)` tuple of bag indices.
+		Rebag(T::AccountId, Option<(BagIdx, BagIdx)>),
```

Comment:
```suggestion
		Rebagged(T::AccountId, Option<(BagIdx, BagIdx)>),
```

If that's actually english.

Reply:
and we shouldn't emit something on Error, so you can remove the `Option<_>`.

Reply:
f58bb3aa0